lib/pbench/server/database/pbench-server.cfg

```diff
@@ -0,0 +1,53 @@
+[DEFAULT]
+# The values here override those in pbench-server-default.cfg.
```

Comment:
This file should have a big comment at the top explaining what it is and how it is used.

Reply:
I just remove this file entirely.